tests/test_script.py

```diff
@@ -24,16 +24,16 @@ def run_node(network, use_tls):
     machine.succeed(f"{tezos_node} config init --data-dir node-dir --network {network}")
     machine.succeed(f"{tezos_node} identity generate 1 --data-dir node-dir")
     machine.succeed(
-        f"{tezos_node} run --data-dir node-dir --rpc-addr 127.0.0.1:8732 "
+        f"{tezos_node} run --data-dir node-dir --rpc-addr :8732 "
         + tls_args
         + "--no-bootstrap-peers --network "
         + network
         + " &"
     )
     tls_endpoint = (
-        " --endpoint https://127.0.0.1:8732/ "
+        " --endpoint https://localhost:8732/ "
```

Comment:
:thinking: why this change?

Reply:
Something has changed in the name resolutions and tests fail if `127.0.0.1` is used with https here :confused: